ocpp/messages.py

```diff
@@ -182,6 +182,53 @@ def get_validator(
     # character. If no encoding is given, reading the schema would fail with:
     #
     #     Unexpected UTF-8 BOM (decode using utf-8-sig):
+    try:
+        with open(path, 'r', encoding='utf-8-sig') as f:
+            data = f.read()
+            validator = Draft4Validator(json.loads(data,
+                                                   parse_float=parse_float))
+            _validators[cache_key] = validator
+        return _validators[cache_key]
+    except FileNotFoundError:
+        return _get_backport_validator(
+            message_type_id,
+            action,
+            ocpp_version,
+            parse_float
+        )
+
+
+def _get_backport_validator(
+        message_type_id: int,
+        action: str,
+        ocpp_version: str,
+        parse_float: Callable = float
+) -> Draft4Validator:
+    """
+    Reads backported schema from disk.
+
+    OCPP 1.6 Improved Security specification has selected Requests and
+    Responses backported from 2.0 to 1.6.
+    """
+    if ocpp_version not in ["1.6"]:
+        raise ValueError
+
+    schemas_dir = 'v' + ocpp_version.replace('.', '')
+
+    schema_name = action
+    if message_type_id == MessageType.CallResult:
+        schema_name += 'Response'
+    elif message_type_id == MessageType.Call:
+        schema_name += 'Request'
```

Comment:
The new schemas for 1.6 still have the same format, so we dont need to add the Request to the name. Which brings to my second point, that I dont thing we need an extra backport code to extract the new schemas. we can simply reuse the old code for that purpose

Reply:
Same as above. I'll remove this logic.